OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -100,6 +400,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                    </list>
                                 </collation>
                                 
+                                <condition key="good">The first leaf and the outer margins of many pages are stained by dirt, e.g. 
+                                <locus target="#25 #30v #31r"/>. The upper edge might have been slightly damaged by fire.
+                                    All edges have been trimmed.</condition>
```

Comment:
This is probably my ignorance, but why are we saying that the edges trimmed? :)

Reply:
I think that they are not necessarily always trimmed and it adds something on the wider practices of manuscript production.

Reply:
To my mind, if we say a page was trimmed, we mean that it was cut down to a smaller size _after_ the manuscript was produced (i.e. written and bound)

Reply:
I agree with @Gnisci. The pages don't seem to have been re-trimmed to a smaller size and the trimming is part of the necessary workflow for making a parchment codex. No examples of non-trimmed Ethiopic parchment codices come to my mind.

Reply:
Thank you! @SusanneHummel what do you think?

Reply:
It is difficult to see from the images. But since the edges are not completely even, i also think they were not trimmed after the ms production. 
I attach you here a clear (at least to me) example of trimmed edges (Ms is from the Veroli collection)


![DSC00015](https://user-images.githubusercontent.com/28757489/97602941-379e8f00-1a0c-11eb-8bc0-e19996dc5758.jpg)